stats/lib/request-helpers.js

```diff
@@ -1,14 +1,14 @@
 import assert from 'http-assert'
 import { json } from 'http-responders'
 import { URLSearchParams } from 'node:url'
-import pg from 'pg'
 
 /** @typedef {import('@filecoin-station/spark-stats-db').Queryable} Queryable */
 
 export const getDayAsISOString = (d) => d.toISOString().split('T')[0]
+export const todayDate = () => new Date(Date.now() - new Date().getTimezoneOffset() * 60000)
 
-export const today = () => getDayAsISOString(new Date())
-export const yesterday = () => getDayAsISOString(new Date(Date.now() - 24 * 60 * 60 * 1000))
+export const today = () => getDayAsISOString(todayDate())
+export const yesterday = () => getDayAsISOString(new Date(todayDate().getTime() - 24 * 60 * 60 * 1000))
```

Comment:
I find this confusing. We need to apply `getTimezoneOffset` only when converting the date-time value into `YYYY-MM-DD` format, could you please move this calculation to `getDayAsISOString`? That way the other places don't need any change.

```js
export const getDayAsISOString = (d) => new Date(d - new Date().getTimezoneOffset() * 60000).toISOString().split('T')[0]

export const today = () => getDayAsISOString(new Date())
export const yesterday = () => getDayAsISOString(new Date(Date.now() - 24 * 60 * 60 * 1000))
```

Thoughts?

Reply:
I had assumed this part had been thoroughly discussed by you two, so I accepted my confusion and thought there would be good reasons.

+1 to what @bajtos said